domain/build.gradle

```diff
@@ -4,6 +4,14 @@ dependencies {
 
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.10.3'
     compile group: 'com.fasterxml.jackson.core', name: 'jackson-databind', version: '2.10.3'
+    compile 'org.springframework.boot:spring-boot-starter-web:2.0.1.RELEASE'
+    compile 'org.springframework.boot:spring-boot-starter-data-jpa:2.0.1.RELEASE'
+   //compile group: 'com.h2database', name: 'h2', version: '1.3.148'
```

Comment:
It's good practice to delete comments like this as they are effectively void and not used, they do not belong in the checked in code as they are bloat. If you want to keep them for yourself to use later I would make a note somewhere